### DISCUSSION TITLE
Add some documentation about a tray application

### DISCUSSION THREAD

**Discussion by sledgehammer999** [Kozea/Radicale#1399] [Ideas]
Hi, I am using radicale for a couple weeks now and it seems to fit my needs.
I am running it on Windows 7, and as you may know, it isn't easy to automatically start the server during login/startup on Windows.

So I quickly coded a tray application in C++ using Qt5. It takes the path to python, radicale.py and radicale's config file. It launches python with the correct arguments and watches the process to tell the status.
I know it is a very simplistic aproach, but AFAIK radicale doesn't provide any special API to monitor it.

Would you be interest in this app? I could make it public.
Also currently I am using the "radish" image taken from your website. I can't find any source on it or any license for it. Is there any .svg of it? What is the licensing on it? Can I use it if I make my application public?

**Comment by liZe** [comment]
> Hi, I am using radicale for a couple weeks now and it seems to fit my needs.

Good news!

> Would you be interest in this app? I could make it public.

Of course! It would be helpful for many Windows users. You can create a project on GitHub (or elsewhere), I'll add a link in the documentation.

> Also currently I am using the "radish" image taken from your website. I can't find any source on it or any license for it. Is there any .svg of it?

Of course, it's in the [gh-pages branch](https://github.com/Kozea/Radicale/tree/gh-pages/css), and it's hand-made. It's public domain, you can use it for your application.
